docs/core/sdk.md

```diff
@@ -1,37 +1,45 @@
 ---
-title: .NET Core SDK Overview
+title: .NET Core SDK overview
 description: Find out about the .NET Core SDK, which is a set of libraries and tools used to create .NET Core projects.
-ms.date: 06/20/2016
+ms.date: 05/13/2019
 ms.technology: dotnet-cli
 ---
-# .NET Core SDK Overview
+# .NET Core SDK overview
 
 .NET Core Software Development Kit (SDK) is a set of libraries and tools that allow developers to create .NET Core applications
 and libraries. This is the package that developers will most likely acquire. 
 
-It contains the following components:
+It contains the following components that are used to build and run applications:
```

Comment:
nit: this should be part of the previous paragraph. The previous sentence ("This is the package...") could also be deleted.